appinventor/appengine/war/index.html

```diff
@@ -28,6 +28,10 @@
       </ul>
     </div>
     <script language="javascript" src="ode/ode.nocache.js"></script>
+    <script type="text/javascript" src="tutorials/tutorial.js"></script>
+    <script type="text/javascript" src="tutorials/Tutorial_DigitalDoodle.js"></script>
+    <script type="text/javascript" src="tutorials/Tutorial_BallBounce.js"></script>
+    <script type="text/javascript" src="tutorials/Tutorial_TalkToMe.js"></script>
```

Comment:
But these should be minimized and combined. Maybe I misunderstood José's comment. Indeed it would be nice if all of the tutorial java script files were combined and minimized at build time (they should be kept separate and not minimized in the source). They just should not be combined with ode.nocache.js